connaisseur/flask_application.py

```diff
@@ -153,6 +153,15 @@ async def __validate_image(type_index, image, admission_request):
     type_, index = type_index
     logging_context.update(image=original_image)
 
+    # https://github.com/sse-secure-systems/connaisseur/issues/820
```

Comment:
I'd still add a bit of context, just the explanation of why and the justification of why it could be ok, shouldn't go here
```suggestion
    # if automatic_unchanged_approval is enabled, admit resource updates
    # if they do not alter the resource's image reference(s)
    # https://github.com/sse-secure-systems/connaisseur/issues/820
```